### PR TITLE
Add support for address answers

### PR DESCRIPTION
**Pull request by MebinAbraham**
### PR Context
Add support for new "Address" answer type introduced in: https://github.com/ONSdigital/eq-questionnaire-runner/pull/261

### Checklist

* [ ] eq-translations updated to support any new schema keys which need translation

### CODE REVIEW

File: schemas/answers/address.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://eq.ons.gov.uk/answers/address.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+      },
+      "q_code": {
```
Comment (LJBabbage) [inline schemas/answers/address.json:10]: q_code? Are we supporting business on this one

Comment (MebinAbraham) [reply schemas/answers/address.json:10]: Runner still support data version 0.0.1 hence `q_code` is kept.

**Review by berroar — COMMENTED** [review]
Does this need a valid test schema to go with it?

**Comment by MebinAbraham** [comment]
> Does this need a valid test schema to go with it?

I'm not sure we need it for new JSON definitions as we don't currently do this for example all answers/question types. We tend to when we have custom python logic to go alongside it. But I guess we have had mixed solutions.

**Review by berroar — APPROVED** [review]

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**